src/maxent_params.cpp

```diff
@@ -259,7 +259,7 @@ void ContiParameters::decompose_covariance_matrix(const alps::params& p){
       }
     }
     for (int i=0; i<ndat(); ++i) {
-      sigma_[i] = std::abs(var(new_ndat_+i))/static_cast<double>(p["NORM"]);
+      sigma_[i] = std::sqrt(std::abs(var(new_ndat_+i)))/static_cast<double>(p["NORM"]);
```

Comment:
oioioioi... looks consistent with the comment on line 221. @ryanlevy please have a close look at this.

Reply:
Looks like there was a `sqrt` in the ALPS version of the code, must have gotten wiped during the port. Apologies about that, I definitely think this should be there.